### DISCUSSION TITLE
xd-AntiSpy 4.12.0

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/xd-AntiSpy#56] [Announcements]
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Just released a private project: xD-AntiSpy 😎! It&#39;s a fork of the original, picking up where the last version left off. ✨ <a href="https://t.co/AfyGzHEtFw">https://t.co/AfyGzHEtFw</a><a href="https://twitter.com/hashtag/xDAntiSpy?src=hash&amp;ref_src=twsrc%5Etfw">#xDAntiSpy</a> <a href="https://twitter.com/hashtag/xpantispy?src=hash&amp;ref_src=twsrc%5Etfw">#xpantispy</a> <a href="https://twitter.com/hashtag/windows11?src=hash&amp;ref_src=twsrc%5Etfw">#windows11</a> <a href="https://twitter.com/hashtag/windows?src=hash&amp;ref_src=twsrc%5Etfw">#windows</a> <a href="https://twitter.com/hashtag/app?src=hash&amp;ref_src=twsrc%5Etfw">#app</a> <a href="https://twitter.com/hashtag/microsoft?src=hash&amp;ref_src=twsrc%5Etfw">#microsoft</a> <a href="https://twitter.com/hashtag/privacy?src=hash&amp;ref_src=twsrc%5Etfw">#privacy</a> <a href="https://t.co/wajGZtGLn5">pic.twitter.com/wajGZtGLn5</a></p>&mdash; Belim (@builtbybel) <a href="https://twitter.com/builtbybel/status/1805906464843694431?ref_src=twsrc%5Etfw">June 26, 2024</a></blockquote>

**I've found the perfect compromise for the user interface by merging the classic and modern versions, resulting in the final form of xd-AntiSpy. It's still small, compact, and intuitive, while maintaining its recognizable  "Xp-AntiSpy" charm. The UI now blends classic user controls with modern elements, landing at a 70% classic and 30% modern mix. But this fusion makes it visually appealing.**

Adresses Issues:
- New and Optimized IPluginLoader which dynamically loads both class library plugins and PowerShell scripts directly into the dropdown menu
- Enhanced Native Plugin Support (the integration of plugins directly in the settings UI via JSON files has been reworked)
- New Modern Info Dialog Window (adapted from the latest modern version)
- New Language Download Window
- Added Turkish Translation (for all translators, new language files will be available soon. Most translations have been automatically applied, but a few strings may need to be removed)
- All Plugins (DLLs) updated with a subtle modern touch
- Numerous minor improvements and code optimizations

![explorer_ROAcAPr9IC](https://github.com/user-attachments/assets/3aef18f1-c5aa-4b67-9345-03bb5c988ead)

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/xd-AntiSpy/releases/tag/4.12.0'>xd-AntiSpy 4.12.0</a>.</em>

**Comment by chap39** [comment]
This is an advertisement that will remain.
If an app offers to get rid of Microsoft advertising, maybe you can get rid of that too?
![Снимок экрана 2024-08-16 163430](https://github.com/user-attachments/assets/02cf1d4e-58cc-4893-8f74-d357006c98ba)
Used app 4.12.0.0.
